lib/datadog/di/contrib/active_record.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Datadog::DI::Serializer.register(condition: lambda { |value| ActiveRecord::Base === value }) \
```

Comment:
Maybe a stupid question, but why not make this condition 
```suggestion
Datadog::DI::Serializer.register(condition: lambda { |value| defined?(ActiveRecord::Base) && ActiveRecord::Base === value }) \
```
so that it can always be loaded, even if ActiveRecord is not loaded?

Reply:
It is a genius idea actually. The reason not to do it is for performance when AR is not loaded (since otherwise every object will have to evaluate this condition) but perhaps it's worthwhile to take the performance hit to permit loading Rails after datadog.